lang/en.js

```diff
@@ -21,14 +21,15 @@ export default {
       "dcTitle": "Title",
       "dcType": "Type of object",
       "dctermsCreated": "Creation date",
-      "dctermsMedium": "Medium",
       "edmCountry": "Providing country",
       "edmDataProvider": "Providing institution",
       "edmRights": "License of the media in this record (unless otherwise specified)",
-      "dcRights": "Rights",
-      "dcDate": "Date",
       "dcPublisher": "Publisher",
       "dcSubject": "Subject",
+      "dcTermsMedium": "Medium",
```

Comment:
This needs to be removed from Lokalise.

Reply:
Done, but won't affect this PR, it'll be gone from the next one.